dsa/sparse_attention.py

```
@@ -0,0 +1,455 @@
+        self.mlp = nn.Sequential(
+            nn.Linear(dim, hidden_dim),
+            nn.GELU(),
+            nn.Dropout(dropout),
+            nn.Linear(hidden_dim, dim),
+            nn.Dropout(dropout),
+        )
+
+    def forward(
+        self,
+        x: torch.Tensor,
+        height: int,
```

Ruff (0.14.8)

Unused method argument: `height`

(ARG002)